### PR TITLE
Fixed Logo Alignment

### LINKED ISSUE #194: Logo alignment in header
Logo in header isn't center aligned
screenshot from 2018-11-26 19-23-42

### PR DESCRIPTION
**Pull request by niranjannitesh**
Fixes #194 

[Preview](https://niteshkumarniranjan.github.io/2012.fossasia.org/)

screenshot from 2018-11-26 19-28-39

### CODE REVIEW

**Review by championpaddler — CHANGES_REQUESTED** [review]
@niteshkumarniranjan  Add preview.

**Comment by niranjannitesh** [comment]
@sk9331657 :+1:

**Review by sarnava1 — APPROVED** [review]

**Review by championpaddler — APPROVED** [review]

File: css/custom.css [hunk 1]
```diff
@@ -269,3 +269,14 @@ a:focus, a:hover {
     color: #fff !important;
     background: #e12b00 !important;
 }
+
```
Comment (championpaddler) [inline css/custom.css:272]: @niteshkumarniranjan  Remove extra line.

**Review by championpaddler — CHANGES_REQUESTED** [review]

**Comment by niranjannitesh** [comment]
@sk9331657 :+1: Done ! :)

**Pull request merged**